content/source/docs/cloud/getting-started/workspaces.html.md

```diff
@@ -34,30 +34,13 @@ To create your first workspace, click the "+ New Workspace" button in the upper
 
 ![new button](./images/work-new-button.png)
 
-On the "Create a new Workspace" page, you need to enter at least two items: a workspace name, and a VCS repository. (You can search for a repo by typing part of its name.) When you've finished, click the "Create Workspace" button.
-
-![fields on new workspace page](../workspaces/images/creating-fields.png)
-
--> **Note:** You can also create workspaces without a VCS repository, but doing so requires a different workflow for Terraform runs. For more information, see [About Terraform Runs in Terraform Cloud](../run/index.html).
-
-### Workspace Name
+On the "Create a new Workspace" page, follow the instructions to configure a VCS repository and a workspace name, then confirm creation with the "Create workspace" button.
 
 A workspace name should tell your colleagues what the workspace is for. Most workspaces are _a particular environment_ of _a particular Terraform configuration,_ so the name should include both the name of the configuration and the name of the environment.
 
 In this example, we're using a configuration named "minimum" and we're deploying it in a production environment, so we named it `minimum-prod`.
```

Comment:
![image](https://user-images.githubusercontent.com/7556325/72306926-8883f900-362d-11ea-8568-ce798c0f414c.png)

I think the image needs to be updated, the workspace names show `migrated-default`, `migrated-first`, `migrated-second` on my version which may be confusing to users looking for `minimum-prod`

Reply:
Ah. Wonder what the deal was with that. 😅 Well, ok!

Reply:
(Of course, I'm planning to dynamite this section anyway. Sigh.)